pkg/apis/3scale/v1alpha1/register.go

```diff
@@ -7,7 +7,7 @@ package v1alpha1
 
 import (
 	"k8s.io/apimachinery/pkg/runtime/schema"
-	"sigs.k8s.io/controller-runtime/pkg/scheme"
+	"sigs.k8s.io/controller-runtime/pkg/runtime/scheme"
```

Comment:
this should stay as `"sigs.k8s.io/controller-runtime/pkg/scheme"`

Reply:
ok